cppguide.html

```diff
@@ -4609,8 +4609,8 @@ <h4 class="stylepoint_subsection">Doxygen Header</h4>
 Every file must contain a doxygen header of the following form. In .cpp files the brief section can be omitted.
 <pre>
 /**
- * @authors     Max Muster
- * @affiliation ANYmal Research Institution
+ * @file        MyClass.hpp
```

Comment:
Just `@file` without postfix will also work by the way. It picks up the filename automagically.

Reply:
```suggestion
 * @file
```

Like this? That would certainly prevent copy/paste mistakes in the future :)

Reply:
exactly. See [doxygen documentation](http://epic-alfa.kavli.tudelft.nl/share/doc/doxygen-1.6.1/html/commands.html#cmdfile).